diag_manager/diag_axis.F90

```diff
@@ -138,9 +138,11 @@ INTEGER FUNCTION diag_axis_init(name, DATA, units, cart_name, long_name, directi
     ENDIF
 
     if (use_modern_diag) then
+      !TODO Passing in the axis_length because of a gnu issue where inside fms_diag_axis_init, the size of DATA
+      !was 2 which was causing the axis_data to not be written correctly...
```

Comment:
If this is a bug with a SPECIFIC gnu version, do we need to fix it or can we just say it's not compatible with that version?  Is this a C5, CI, or dev box compiler?

Reply:
The version of gnu is what we used in the CI
I can also reproduce in the dev box.